pages/account/deposit.vue

```diff
@@ -291,10 +291,20 @@ const { balanceOf } = useLikeCoinContract()
 
 const stakeAmount = ref(0)
 const withdrawAmount = ref(0)
-const isAutoRestakeEnabled = useStorage(`${cacheKeyPrefix}-deposit-autorestake`, true)
+const isAutoRestakeEnabledStorage = useStorage(`${cacheKeyPrefix}-deposit-autorestake`, true)
+const isAutoRestakeEnabled = ref(true)
 const isLoading = ref(false)
 const error = ref<string | null>(null)
 
+watch(isAutoRestakeEnabled, (newValue) => {
+  isAutoRestakeEnabledStorage.value = newValue
+})
+
+onMounted(() => {
+  // HACK: Mitigate USwitch initial value out sync wih v-model issue
+  isAutoRestakeEnabled.value = isAutoRestakeEnabledStorage.value
+})
+
```

Comment:
How about using computed with setter?
```suggestion
// HACK: Mitigate USwitch initial value out sync with v-model issue
const isAutoRestakeEnabled = computed({
  get: () => isAutoRestakeEnabledStorage.value,
  set: (value) => {
    isAutoRestakeEnabledStorage.value = value
  },
})
```

Reply:
Seems a value change needs to be triggered on mounted to prevent SSR issue